docs/BuildONNX.md

```diff
@@ -2,7 +2,7 @@
 
 # Installing `third_party ONNX` for Backend Tests or Rebuilding ONNX Operations
 
-Backend tests are triggered by `make check-onnx-backend` in the build directory and require a few preliminary steps to run successfully. Similarly, rebuilding the ONNX operations in ONNX-MLIR from their ONNX descriptions is triggered by `make OMONNXOpsIncTranslation`.
+Backend tests are triggered by `ninja check-onnx-backend` in the build directory and require a few preliminary steps to run successfully. Similarly, rebuilding the ONNX operations in ONNX-MLIR from their ONNX descriptions is triggered by `ninja OMONNXOpsIncTranslation`.
```

Comment:
Could we keep both ninja and make in document?

Reply:
I am not sure I understand why we want to keep both. Other documents like BuildOnWindows.md and BuildOnLinuxOSX.md uses only ninja.

Reply:
Because not everybody is familiar with ninja. And most systems don't come with ninja pre-installed. Other documents should have been updated with both.

Reply:
Sorry I am not sure if the change worth it. I will drop the pr. Thank you for looking into it. Appreciate your time.